imperfect-perfect.tex

```diff
@@ -433,6 +443,76 @@ \section{Interlude on perfectoid fields}
 Using Lemma~\ref{L:enough phi-invariants}, this follows as in the proof of \cite[Theorem~6.3.12]{kedlaya-liu1}.
 \end{proof}
 
+\begin{remark}
+Throughout this remark, assume that $A$ is a field.
+In this setting, the concept of $B$-pairs was introduced by Berger \cite{berger-b-pairs}
+in a purely algebraic fashion,
+without reference to the schemes defined in Definition~\ref{D:B-pairs}. Therein,
+the ring $\bB_{e,L,A}$ appears in connection with Fontaine's crystalline period ring
+$\bB_{\mathrm{crys}}$.
+
+The scheme $X_{L,A}$ introduced by Fargues--Fontaine is in some sense a ``complete curve'': in particular, it is a regular one-dimensional noetherian scheme.
+The space $P_{L,A,n}$ constitutes the sections of the $n$-th power of a certain ample line bundle on this scheme. This scheme admits something resembling an analytification in the category of adic spaces, in that there is a morphism into it from an adic space built out of the rings $\tilde{\calR}^{[s,r]}_{L,A}$, the pullback along which induces an equivalence of categories of coherent sheaves by analogy with Serre's GAGA theorem in complex algebraic geometry. This adic space in turn admits an infinite cyclic \'etale cover which is
+a ``quasi-Stein space'' whose global sections are the ring $\bigcap_{r>0} \tilde{\calR}^r_{L,A}$, on which the deck transformations act via the powers of $\varphi$.
```

Comment:
I don't think there's an issue here. Perhaps a decent analogy is: projective n-space is, in some suitable sense, the quotient of affine (n+1)-space by the action of G_m; but you make it by taking Proj of a graded ring in which each graded piece is an eigenspace of G_m for a certain *nontrivial* character.

Let me know if you want me to put a version of this text in, or leave it alone.